source/libraries/NetworkConfigurations.ts

```diff
@@ -18,7 +18,7 @@ export const networkConfigurations: NetworkConfigurations = {
         host: "ropsten.ethereum.nodes.augur.net",
         port: 80,
         privateKey: process.env.ROPSTEN_PRIVATE_KEY,
-        gasPrice: new BN(20)
```

Comment:
should be multiplied by 10^9 instead of 10^6?

Reply:
Same error as with Ropsten ,good call.

Reply:
this has been updated in a later patch

Reply:
In a later patch that is forthcoming? I see same issue on master